.travis.yml

```diff
@@ -64,6 +66,18 @@ matrix:
       script:
         - python3 setup.py bdist_wheel
       deploy: *_deploy
+    - name: 'Push image to Docker Hub'
+      python: 3.5
+      stage: deploy
+      script:
+        - DOCKER_PUSH_LATEST=true make docker-push
```

Comment:
@rporres Currently, we are using DockerHub Automated Builds. Shall I cancel them on DockerHub once we push here?

Reply:
Yes. This is the standard way of creating containers across the different company repositories.